iRonCub-Mk3/hardware/motorControl/left_arm-eb1-j0_1-mc.xml

```diff
@@ -118,6 +118,19 @@
         <param name="deadZone">                 0.0049              0.0049              </param>
     </group>
 
+    <!-- kalman filter parameters - vel & acc estimation -->
+    <group name="KALMAN_FILTER">
+        <param name="kalmanFilterEnabled">          1              1            1          </param>
+        <param name="x0">                           0              0            0          </param>
+        <param name="x1">                           0              0            0          </param>
+        <param name="x2">                           0              0            0          </param>
+        <param name="Q0">                           0.0001         0.0001       0.0001     </param>
+        <param name="Q1">                           0.01           0.01         0.01       </param>
+        <param name="Q2">                           10             10           10         </param>
+        <param name="R">                            0.001          0.001        0.001      </param>
+        <param name="P0">                           0.000099       0.000099     0.000099   </param>
```

Comment:
I think the number of columns corresponds to how many joints this config file is controlling.

For instance, this file `left_arm-eb1-j0_1-mc.xml` concerns 2 joints (`shoulder_pitch` and `shoulder_roll`). So this part should have 2 columns (I think).

Similar logic applies to the rest of the files.

Reply:
Yes, I made a mistake here, I will fix it.